torch/serialization.py

```diff
@@ -17,7 +17,7 @@
 
 DEFAULT_PROTOCOL = 2
 
-LONG_SIZE = struct.Struct('=l').size
+LONG_SIZE = struct.Struct('=q').size
```

Comment:
why? It's `long long` and not `long`. The whole point of this variable is to encode the `long` size, because it varies between systems